tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php

```diff
@@ -1478,4 +1479,23 @@ public function testPostEntryWhenFetchFails()
         $this->assertSame('email_tracking.pdf', $content->getTitle());
         $this->assertSame('example.com', $content->getDomainName());
     }
+
+    public function testEntryDeleteTagLink()
+    {
+        $this->logInAs('admin');
+        $client = $this->getClient();
+
+        $em = $client->getContainer()->get('doctrine.orm.entity_manager');
+        $entry = $em->getRepository('WallabagCoreBundle:Entry')->findByUrlAndUserId('http://0.0.0.0/entry1', $this->getLoggedInUserId());
+        $tag = $entry->getTags()[0];
+
+        $crawler = $client->request('GET', '/view/' . $entry->getId());
+
+        // As long as the deletion link of a tag is following
+        // a link to the tag view, we take the second one to retrieve
+        // the deletion link of the first tag
+        $link = $crawler->filter('body div#article div.tools ul.tags li.chip a')->extract('href')[1];
+
+        $this->assertSame(sprintf('/remove-tag/%s/%s', $entry->getId(), $tag->getId()), $link);
```

Comment:
Why don't you follow the link to ensure it returns a 301?

Reply:
Do you want a complete test in the same PR or can we delay it to another PR?

Reply:
As you want 🎅